.gitignore

```diff
@@ -23,6 +23,9 @@ output/
 .idea/**/tasks.xml
 .idea/dictionaries
 
+# BBEdit projects
+**/*.bbprojectd
```

Comment:
It also seems to be missing `.DS_Store`, which found its way into `gruberwine/assets`, please remove it. (It is listed in the main repo.)

Reply:
Done.

Reply:
The `assets/.DS_Store` file is still there, you’ll need to `git rm` it.

Reply:
Thanks. Done.